server/tests/src/test/java/org/infinispan/server/resilience/GracefulShutdownRestartIT.java

```diff
@@ -60,4 +64,49 @@ public void testGracefulShutdownRestart() {
          assertEquals(String.format("v%03d", i), hotRod.get(String.format("k%03d", i)));
       }
    }
+
+   @Test
+   public void testCrashBeforeFinishedStarting() throws Exception {
+      ConfigurationBuilder builder = new ConfigurationBuilder();
+      builder.clustering().cacheMode(CacheMode.DIST_SYNC).persistence().addSingleFileStore().segmented(false);
```

Comment:
Why do we need a cache with persistence here?

Reply:
Too fast in the copy-paste :sweat_smile: